# Type: docs
# Subtype: D3
# English Summary: Add copyright notice
# Chinese Summary: 添加版权声明
# Entry Number: 5

# === Git Diff Content ===

diff --git a/querydsl-core/src/main/java/com/mysema/query/EmptyMetadata.java b/querydsl-core/src/main/java/com/mysema/query/EmptyMetadata.java
index 34f6244cf..80b24d0b0 100644
--- a/querydsl-core/src/main/java/com/mysema/query/EmptyMetadata.java
+++ b/querydsl-core/src/main/java/com/mysema/query/EmptyMetadata.java
@@ -1,3 +1,16 @@
+/*
+ * Copyright 2012, Mysema Ltd
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 package com.mysema.query;
 
 import java.util.Collections;
